packages/react/src/components/Card/CardSection.tsx

```diff
@@ -3,26 +3,24 @@ import cl from 'clsx/lite';
 import type { HTMLAttributes } from 'react';
 import { forwardRef } from 'react';
 
-export type CardMediaProps = {
+export type CardSectionProps = {
   /**
    * Change the default rendered element for the one passed as a child, merging their props and behavior.
    * @default false
    */
   asChild?: boolean;
 } & HTMLAttributes<HTMLDivElement>;
 
-export const CardMedia = forwardRef<HTMLDivElement, CardMediaProps>(
-  ({ asChild, className, ...rest }, ref) => {
+export const CardSection = forwardRef<HTMLDivElement, CardSectionProps>(
+  function CardSection({ asChild, className, ...rest }, ref) {
     const Component = asChild ? Slot : 'div';
 
     return (
       <Component
-        className={cl(`ds-card__media`, className)}
+        className={cl(`ds-card__section`, className)}
         ref={ref}
         {...rest}
       />
     );
   },
 );
```

Comment:
Should this be called "section" when we don't render a `section` element?
I would suggest something along the lines of `Card.Part` or `Card.Block`? 
Not really sure here, but I think we should find another name, so we are consistent with our API

Reply:
Yeah, I agree - was a bit unsure about the naming. I like `.Part` ☺️